templates/CRM/Contact/Form/Task/Email.hlp

```diff
@@ -40,6 +41,7 @@
 {/htxt}
 {htxt id="id-token-subject"}
 <p>{ts}Use a token in the Subject when you are sending mail to a number of recipients and want to include their name and / or other values from their contact record automatically in the subject line.{/ts}</p>
+<p>{ts 1=$tokenDocs}%1{/ts}</p>
```

Comment:
There's a known issue with `{docUrl}` and translation (https://lab.civicrm.org/dev/translation/-/issues/61) which makes this difficult, but in this line all translators will see is `%1`. There's are some suggestions in that ticket for how to use docURL, although seems a bit of an open question still.

Reply:
@demeritcowboy thanks for the heads up on that core issue. Any advise on how to change this PR to be acceptable? By the way, this tokenDocs pattern was already in place, I just moved it to this help file so it could be re-used.

Reply:
@demeritcowboy tested and works fine, ready for review Mr. D.

Reply:
![Screenshot_20210511_151546](https://user-images.githubusercontent.com/58866555/117762048-bf8f0c80-b26b-11eb-95f9-fab7d17f9f78.png)